package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.3.3",
   "description": "Random data generator with localization",
   "main": "dist/fakerator.js",
+  "types": "dist/fakerator.d.ts",
   "directories": {
     "test": "test"
   },
   "scripts": {
     "prebuild": "npm run test",
-    "build": "webpack --config config/webpack.build.config.js",
+    "copytype": "cp fakerator.d.ts dist/fakerator.d.ts",
```

Comment:
Would be better an OS-dependent solution to copy file.

Reply:
Do you have a solution in mind?
for me cp is well supported:

[All UNIX systems. Windows Server 2012. Windows 8.1. Windows Server 2012 R2. Windows 10. Windows Server 2016. Windows Server 2019.](https://www.mkssoftware.com/docs/man1/cp.1.asp)

Reply:
Try to use it: https://github.com/calvinmetcalf/copyfiles

Reply:
@icebob done !